### DISCUSSION TITLE
[Events] OnPlayerKillAnimal/OnPlayerKillZombie

### DISCUSSION THREAD

**Discussion by Fanya-dev** [SmartlyDressedGames/Unturned-3.x-Community#4268] [Ideas]
It's very strange that there are no these events, I'd really like to see them. Or did I not look hard enough?

**Comment by cry-1337** [comment]
i think u need to use OnPlayerStatChanged. or patch with harmony

**Comment by Corbyncc** [comment]
If you are using OpenMod (you should be) there is the `Unturned(Animal/Zombie)DeadEvent`

or

`DamageTool.damageAnimalRequested` and `DamageTool.damageZombieRequested` and do the calculations yourself on if its a kill or not.

or

Patch `Zombie.tellDead` and `Animal.tellDead`
